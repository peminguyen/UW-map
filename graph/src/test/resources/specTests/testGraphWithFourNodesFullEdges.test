CreateGraph graphWithFourNodesFullEdges

AddNode graphWithFourNodesFullEdges A
AddNode graphWithFourNodesFullEdges B
AddNode graphWithFourNodesFullEdges C
AddNode graphWithFourNodesFullEdges D

AddEdge graphWithFourNodesFullEdges A B eAB
AddEdge graphWithFourNodesFullEdges A C eAC
AddEdge graphWithFourNodesFullEdges A D eAD

AddEdge graphWithFourNodesFullEdges B A eBA
AddEdge graphWithFourNodesFullEdges B C eBC
AddEdge graphWithFourNodesFullEdges B D eBD

AddEdge graphWithFourNodesFullEdges C A eCA
AddEdge graphWithFourNodesFullEdges C B eCB
AddEdge graphWithFourNodesFullEdges C D eCD

AddEdge graphWithFourNodesFullEdges D A eDA
AddEdge graphWithFourNodesFullEdges D B eDB
AddEdge graphWithFourNodesFullEdges D C eDC

ListNodes graphWithFourNodesFullEdges
ListChildren graphWithFourNodesFullEdges A
ListChildren graphWithFourNodesFullEdges B
ListChildren graphWithFourNodesFullEdges C
ListChildren graphWithFourNodesFullEdges D