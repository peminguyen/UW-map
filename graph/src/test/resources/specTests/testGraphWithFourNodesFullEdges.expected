created graph graphWithFourNodesFullEdges

added node A to graphWithFourNodesFullEdges
added node B to graphWithFourNodesFullEdges
added node C to graphWithFourNodesFullEdges
added node D to graphWithFourNodesFullEdges

added edge eAB from A to B in graphWithFourNodesFullEdges
added edge eAC from A to C in graphWithFourNodesFullEdges
added edge eAD from A to D in graphWithFourNodesFullEdges

added edge eBA from B to A in graphWithFourNodesFullEdges
added edge eBC from B to C in graphWithFourNodesFullEdges
added edge eBD from B to D in graphWithFourNodesFullEdges

added edge eCA from C to A in graphWithFourNodesFullEdges
added edge eCB from C to B in graphWithFourNodesFullEdges
added edge eCD from C to D in graphWithFourNodesFullEdges

added edge eDA from D to A in graphWithFourNodesFullEdges
added edge eDB from D to B in graphWithFourNodesFullEdges
added edge eDC from D to C in graphWithFourNodesFullEdges

graphWithFourNodesFullEdges contains: A B C D
the children of A in graphWithFourNodesFullEdges are: B(eAB) C(eAC) D(eAD)
the children of B in graphWithFourNodesFullEdges are: A(eBA) C(eBC) D(eBD)
the children of C in graphWithFourNodesFullEdges are: A(eCA) B(eCB) D(eCD)
the children of D in graphWithFourNodesFullEdges are: A(eDA) B(eDB) C(eDC)